pkg/signoz/signoz.go

```diff
@@ -13,9 +14,10 @@ import (
 )
 
 type SigNoz struct {
-	Cache    cache.Cache
-	Web      web.Web
-	SQLStore sqlstore.SQLStore
+	Cache     cache.Cache
+	Web       web.Web
+	APIServer apiserver.APIServer
```

Comment:
```suggestion
	APIServer apiserver.APIServer
```

Don't need this anymore